### PR TITLE
show alert on disabled account

### PR DESCRIPTION
**Pull request by ywrsusan**

### CODE REVIEW

File: pybossa/view/account.py [hunk 1]
```diff
@@ -188,6 +191,15 @@ def signin():
         # User already signed in, so redirect to home page
         return redirect_content_type(url_for("home.home"))
 
+def disable_redirect(brand=None):
```
Comment (jayflo) [inline pybossa/view/account.py]: You shouldn't need to pass the `brand` here.  You can just fetch it in this function `brand = current_app.config['BRAND']`

Comment (dchhabda) [reply pybossa/view/account.py]: agree w/ jaret

File: test/test_web.py [hunk 2]
```diff
@@ -1428,6 +1428,27 @@ def test_05_test_signout_json(self):
         err_msg = "success message missing"
         assert data['status'] == 'success', err_msg
 
+    @with_context
+    @with_context_settings(ENABLE_TWO_FACTOR_AUTH=True)
+    def test_06_test_disabled_user_signin_1(self):
+        res = self.register()
+        res = self.signout()
```
Comment (dchhabda) [inline test/test_web.py]: is this admin signin to update user's state?

Comment (fiorda) [reply test/test_web.py]: it's just to create the user

**Pull request merged**